unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -25,7 +25,7 @@
 )
 
 if TYPE_CHECKING:
-    from astrapy.db import AstraDBCollection
+    import astrapy
```

Comment:
To help with typing, can we be more explicit in what's being imported? 
```suggestion
    from astrapy import Collection as AstraDBCollection
```

Reply:
Updated!